frontend/test/metabase/scenarios/native/native_subquery.cy.spec.js

```diff
@@ -263,4 +267,33 @@ describe("scenarios > question > native subquery", () => {
       cy.contains("41");
     });
   });
+
+  it.skip("should be able to reference a nested question (metabase#25988)", () => {
+    const questionDetails = {
+      name: "Nested GUI question",
+      query: {
+        "source-table": "card__1",
+        limit: 2,
+      },
+    };
+
+    cy.createQuestion(questionDetails).then(
+      ({ body: { id: nestedQuestionId } }) => {
+        const tagID = `#${nestedQuestionId}`;
+
+        startNewNativeQuestion();
```

Comment:
There is existing `openNativeEditor` helper that does the same

Reply:
It doesn't.
The other one uses the UI and is much slower. This one generates a base64 string from an ad-hoc query and goes straight to the native editor.